apps/studio/components/interfaces/Connect/ConnectionPanel.tsx

```diff
@@ -0,0 +1,283 @@
+              {notice?.map((text: string) => (
+                <p key={text} className="text-xs text-foreground-lighter">
+                  {text}
+                </p>
+              ))}
+            </div>
+          )}
+          {parameters.length > 0 && <ConnectionParameters parameters={parameters} />}
+        </div>
+      </div>
+      <div className="flex flex-col items-end">
+        <div className="flex flex-col -space-y-px w-full">
+          {type !== 'session' && (
+            <>
+              <div className="relative border border-muted px-5 flex items-center gap-3 py-3 first:rounded-t last:rounded-b h-[58px]">
+                <div className="absolute top-2 left-2.5">
+                  {type === 'transaction' ? <TransactionIcon /> : <DirectConnectionIcon />}
+                </div>
+                <div className="flex flex-col pl-[52px]">
+                  <span className="text-xs text-foreground">
+                    {type === 'transaction'
+                      ? 'Suitable for a large number of connected clients'
+                      : 'Suitable for long-lived, persistent connections'}
+                  </span>
+                </div>
+              </div>
+              <div className="border border-muted px-5 flex items-center gap-3 py-3 first:rounded-t last:rounded-b h-[58px]">
+                <div className="flex flex-col pl-[52px]">
+                  <span className="text-xs text-foreground">
+                    {type === 'transaction'
+                      ? 'Pre-warmed connection pool to Postgres'
+                      : 'Each client has a dedicated connection to Postgres'}
+                  </span>
+                </div>
+              </div>
+            </>
+          )}
+
+          <div className="border border-muted px-5 flex gap-7 items-center py-3 first:rounded-t last:rounded-b">
+            <div className="flex items-center gap-2">
+              <IPv4StatusIcon active={ipv4Status.type === 'success'} />
+            </div>
+            <div className="flex flex-col">
+              <span className="text-xs text-foreground">{ipv4Status.title}</span>
+              {ipv4Status.description && (
+                <span className="text-xs text-foreground-lighter">{ipv4Status.description}</span>
+              )}
+              {/* {ipv4Status.type === 'success' && type !== 'transaction' && (
+                  <span className="text-xs text-foreground-lighter">
+                    Use Direct Connection if on a IPv6 network
+                  </span>
+                )} */}
+              {ipv4Status.type === 'error' && (
+                <span className="text-xs text-foreground-lighter">
+                  Use Session Pooler if on a IPv4 network or purchase IPv4 addon
+                </span>
+              )}
+              {ipv4Status.link && (
+                <div className="mt-2">
+                  <Button className="" type="default" size="tiny">
+                    <a
```

Comment:
use <Link> component - otherwise routing is wrong (rendered URL will omit /dashboard, can verify this behaviour on the preview)